packages/frontend/app/styles/components/user-guide-link.scss

```diff
@@ -0,0 +1,27 @@
+@use "../ilios-common/mixins" as cm;
+@use "../ilios-common/colors" as c;
+@use "../mixins" as m;
+
+.user-guide-link {
+  @include m.header-menu;
+
+  @include cm.for-tablet-and-up {
+    margin-left: 0.25rem;
+    margin-right: 0.5rem;
+  }
+
+  @include cm.for-phone-only {
+    /* stylelint-disable property-disallowed-list */
+    font-size: 3.5vw;
```

Comment:
I don't love using this disallowed property, Could this be done by controlling width and height of the icon?

Reply:
I removed this disallowed property and used height/width instead.